cmd/autoscaler/main.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/knative/pkg/version"
 	"github.com/knative/serving/pkg/apis/serving"
 	"github.com/knative/serving/pkg/autoscaler"
+	config "github.com/knative/serving/pkg/autoscaler/config"
```

Comment:
you dont need to name it explicitly if the directory is already named that.

Reply:
Done.